### ISSUE TITLE
[Question] Automatically add row for 2weeks after and remove rows of yesterday

### ISSUE THREAD

**Issue by rathod-sahaab** [typeorm/typeorm#6099]
**Issue type:**
[x] question

**Database system/driver:**
[x] `postgres`

**TypeORM version:**
[x] `0.2.24` 

**TLDR**
As clear from the title I am making a game room app, Using NodeJS and TypeORM for the backend.

Some preface:
- Multiple shops
- A shop runs on multiple dates and has multiple SEATS
- You can book say in advance 2 weeks
My current "solution":
- An array of 5-minute long time slots containing number SEATS occupied
 - If the shop opens for 2 hrs
  - [ 0, 1, 5, 1, ...24 elements]
  - Where index I is the Ith 5minute of the day

| ID | DATE     | SHOP | filledSeats                   |
|----|----------|------|-------------------------------|
|  1 | 5/5/2020 | 21   | [ 0, 1, 5, 1, ...24 elements] |

```ts
@Entity("occupancies")
export class Occupancy extends BaseEntity {
    @PrimaryGeneratedColumn("uuid")
    id: string;

    @Column()
    date: Date;
 
    @ManyToOne(()=>Shop, (shop: Shop) => shop.occupancies)
    shop: Shop;

    @Column({array: true})
    filled: number[];
}
```

I want to automatically delete yesterday's entries and add rows for 2 weeks later. How can I do that? Also feel free to suggest other approaches for the problem. Thanks for the awesome tool!!

**Comment by surajrana52** [comment]
This is not relevant to the Typeorm, please use stackoverflow.com for this.
